src/components/Events/Fall_GM/style.scss

```diff
@@ -1,62 +1,167 @@
-#time-to-gm-headline {
-  font-size: 3em;
-  text-align: center;
-}
-#time-to-gm-headline-now {
-  font-size: 3em;
-  text-align: center;
-}
-#countdown-wrapper {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
-  margin: auto;
-  margin-top: -125px;
-  width: 75%;
+#fall-gm-page{
+  #time-to-gm-wrapper{
+    #time-to-gm-headline {
+      font-size: 3em;
+      text-align: center;
+    }
+    #countdown-wrapper {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: center;
+      margin: auto;
+      margin-top: -125px;
+      width: 75%;
+
+      .countdown-cards {
+        color: white;
+        margin: 40px;
+        .square-background {
+          position: relative;
+          top: 140px;
+          z-index: -1;
+          height: 150px;
+          width: 150px;
+          border-radius: 15px;
+          transform: rotate(45deg);
+          background-color: $acm-cobalt;
+        }
```

Comment:
This is kind of a horrible hack, but it's analogous to the way our <Banner> component works. With the media queries this displays decently on both web and mobile clients

Reply:
I hate css, looks pretty tho

Reply:
Going to resolve then if no issues with the css?